l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
I only used the _order property in the l10n_br_account with: "date_invoice desc, internal_number desc". Because if used the date_hour_invoice there will not be an sort, I think it will became confuse.